internal/backend/crypto/age/ssh.go

```diff
@@ -69,6 +66,24 @@ func (a *Age) getSSHIdentities(ctx context.Context) (map[string]age.Identity, er
 	return ids, nil
 }
 
+func getSSHDir() (string, error) {
+	preferredPath := os.Getenv("GOPASS_SSH_DIR")
+	sshDir := filepath.Join(preferredPath, ".ssh")
+	if fsutil.IsDir(sshDir) {
```

Comment:
I think we should also check if `preferredPath` is not empty here. Otherwise we might end up with `.ssh` (i.e. .ssh in the current pwd) w/o users asking for it.

Reply:
thanks @dominikschulz , i've added the check